public/pages/CreateDetector/components/DefineDetector/components/DetectionRules/DetectionRules.tsx

```diff
@@ -63,9 +64,17 @@ export const DetectionRules: React.FC<DetectionRulesProps> = ({
     <EuiPanel style={{ paddingLeft: '0px', paddingRight: '0px' }}>
       <EuiAccordion
         buttonContent={
-          <EuiTitle>
-            <h4>{`Detection rules (${enabledRulesCount} selected)`}</h4>
-          </EuiTitle>
+          <>
+            <EuiTitle>
+              <h4>{`Detection rules (${enabledRulesCount} selected)`}</h4>
+            </EuiTitle>
+            <EuiText size="s">
+              <span style={{ color: 'grey', fontWeight: 200, fontSize: '15px' }}>
```

Comment:
Is it necessary to use `span` here and configure the font size? Typically, we try to stick to the `s`, `m`, etc. options that are built into the `EuiText` and other text-based components to help keep the font sizes consistent throughout the pages.

Reply:
I had to use the code above, to have it consistent between ContentPanel subTitleText prop implementation used on the page and Detection rules subsection which uses accordion and EuiPanel. EuiPanel has no option for subtitle text, so I reused the code from ContentPanel implementation. Otherwise it would differ between the subsections of Define Detector page.

Reply:
possibly in the EuiText you could try color="subdued" instead of using span?

Reply:
I agree. EuiText small and color="subdued" should be able to achieve the same without defining inline styles.